amgut/db/patches/0029.sql

```diff
@@ -0,0 +1,174 @@
+-- Oct 27, 2015
+-- Add way to track what surveys are duplicates for participant consent
+
+--Add timestamp for when a kit is registered
+ALTER TABLE ag.ag_kit ADD registered_on timestamp DEFAULT current_timestamp;
+
+--Add table to track duplicates
+CREATE TABLE ag.duplicate_consents ( 
```

Comment:
agreagate PK would work best, with a unique on duplicate survey. Will do that now.